tests/python/contrib/test_verilator/infrastructure.py

```diff
@@ -66,10 +68,43 @@ def offload(mod):
     return mod
 
 
+def verilator_app_path():
```

Comment:
I don't think it's actually running in the CI (see https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-7286/6/pipeline/255#step-340-log-91), would be better if we could enable testing this.

Reply:
Hey @liangfu

Yeah that is the plan, once we merge this. The idea is to finally update that PR that we have open for CI.